src/frontends/tensorflow_common/src/helper_transforms/const_to_result_remover.cpp

```diff
@@ -20,11 +20,9 @@ bool ConstToResultRemover::run_on_model(const std::shared_ptr<ov::Model>& m) {
     // no need to apply it for sub-graphs!
     ResultVector results_to_remove;
     // look for isolated UnsupportedConst->Result sub-graphs to remove
-    // also, find isolated Constant->Result sub-graphs to remove
     for (const auto& result : m->get_results()) {
         auto unsupported_const = as_type_ptr<UnsupportedConstant>(result->get_input_node_shared_ptr(0));
-        auto const_node = as_type_ptr<v0::Constant>(result->get_input_node_shared_ptr(0));
-        if (unsupported_const || const_node) {
```

Comment:
we have a model that contains multiple isolated Constants (tens of them) that impact compilation times of the plugin.
This was a reason to delete such isolated constants.